lib/manageiq/providers/workflows/engine.rb

```diff
@@ -68,17 +69,17 @@ def self.set_floe_runner
               "task_service_account" => ENV.fetch("AUTOMATION_JOB_SERVICE_ACCOUNT", nil)
             }.merge(floe_runner_settings.kubernetes)
 
-            Floe.set_runner("docker", "kubernetes", options)
+            Floe::Runner.register_scheme("docker", "kubernetes", options)
```

Comment:
Should be (if floe@master has been released):

```suggestion
            Floe::ContainerRunner.set_runner("kubernetes", options)
```